### PR TITLE
Increase testing quality 

### PR DESCRIPTION
**Pull request by Maxwell-Baird**
## Description

Add more tests to check details about the responses back from the api.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] This added to the test

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by rubyr** [comment]
Glad we're getting things published!

File: dungeon/views.py [hunk 1]
```diff
@@ -7,25 +7,26 @@
...
 
     elif request.method == 'POST':
         npc_data = JSONParser().parse(request)
         npc_serializer = NpcSerializer(data=npc_data)
         if npc_serializer.is_valid():
             npc_serializer.save()
-            return JsonResponse(npc_serializer.data, status=status.HTTP_201_CREATED)
-        return JsonResponse(npc_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+            return Response(npc_serializer.data, status=status.HTTP_201_CREATED)
+        return Response(npc_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```
Comment (KirbyDD) [inline dungeon/views.py:26]: What is the difference between the two?

Comment (Maxwell-Baird) [reply dungeon/views.py:26]: Jsonresponse when look at by testing only returns the status code and a reference to the actual json. Response returns that actually data

Comment (KirbyDD) [reply dungeon/views.py:26]: ahh thank you for explaining

**Pull request merged**